cmd/migrate/main.go

```diff
@@ -0,0 +1,40 @@
+package migrate
+
+import (
+	"github.com/aliazizii/IMDB-In-Golang/config"
+	"github.com/aliazizii/IMDB-In-Golang/internal/auth"
+	"github.com/aliazizii/IMDB-In-Golang/internal/database"
+	"github.com/aliazizii/IMDB-In-Golang/internal/model"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func main(cfg *config.Config) {
+	db, err := database.New(cfg.DB)
+	if err != nil {
+		logrus.Fatal(err)
+	}
+
+	err = db.AutoMigrate(&model.Movie{}, &model.User{}, &model.Comment{}, &model.Vote{})
+	if err != nil {
+		logrus.Fatal(err)
+	}
+
+	db.Save(&model.User{
```

Comment:
result of this Save is ignored. It's better to log it to know if it failed.

Also, it's not very practical to save admin credentials this way.